airbyte-integrations/connectors/source-news-api/source_news_api/spec.yaml

```diff
@@ -0,0 +1,181 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/news-api
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: News Api Spec
+  type: object
+  required:
+    - api_key
+    - country
+    - category
+    - sort_by
+  additionalProperties: true
+  properties:
+    api_key:
+      type: string
+      description: API Key
+      airbyte_secret: true
+      order: 0
+    search_query:
+      type: string
+      description: |
+        Search query. See https://newsapi.org/docs/endpoints/everything for 
+        information.
+      order: 1
+    search_in:
+      type: array
+      description: |
+        Where to apply search query. Possible values are: title, description,
+        content.
+      items:
+        type: string
+        enum:
+          - title
+          - description
+          - content
+      order: 2
+    sources:
+      type: array
+      description: |
+        Identifiers (maximum 20) for the news sources or blogs you want
+        headlines from. Use the `/sources` endpoint to locate these
+        programmatically or look at the sources index.
```

Comment:
Please add the link to https://newsapi.org/sources

Reply:
Done